code/lib/types/src/modules/core-common.ts

```diff
@@ -168,6 +168,7 @@ export interface BuilderOptions {
   cache?: FileSystemCache;
   configDir: string;
   docsMode?: boolean;
+  env: Record<string, string>;
```

Comment:
Either the [docs](https://storybook.js.org/docs/react/configure/environment-variables#using-storybook-configuration) are wrong or this has to be a function, which returns a Record

Reply:
Also, does `env` has to be required?

Reply:
I will apply the changes required for Angular as soon as this is resolved

Reply:
Thanks you're totally right, not sure what I was thinking.  🤦  I've made it optional and fixed the signature.  I think angular won't be an issue now.